case-study.Rmd

```diff
@@ -0,0 +1,467 @@
+---
+title: "ADAGE signature analysis - case study"
+author: "Jie Tan"
+date: "`r Sys.Date()`"
+output:
+  rmarkdown::html_vignette:
+    fig_width: 7
+    fig_height: 7
+vignette: >
+  %\VignetteIndexEntry{User input example}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, warning = FALSE, message = FALSE)
+```
+
+This script shows how we analyze the *∆anr* mutation dataset used as a case study
+in the ADAGE signature analysis paper. 
+
+# Data preparation
+Load in required libraries.
+
+```{r}
```

Comment:
Interesting, I did not notice this difference before. The bracket is definitely introduced by `knitr`.